plugins/osdn/node.go

```diff
@@ -0,0 +1,140 @@
+package osdn
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	log "github.com/golang/glog"
+
+	"github.com/openshift/openshift-sdn/pkg/netutils"
+	"github.com/openshift/openshift-sdn/plugins/osdn/api"
+
+	osclient "github.com/openshift/origin/pkg/client"
+	osapi "github.com/openshift/origin/pkg/sdn/api"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/container"
+	kexec "k8s.io/kubernetes/pkg/util/exec"
+	kubeutilnet "k8s.io/kubernetes/pkg/util/net"
+)
+
+type OsdnNode struct {
+	multitenant        bool
```

Comment:
I could, but after creation time, we'd never actually use the plugin name for anything other than checking if it's multitenant, so that kind of seems like a waste. (Well, ok, plugin.Name() would make more sense if we had the pluginName rather than multitenant, I guess...)

So should I change it or not?